modules/ROOT/pages/mule-message-encryption-processor.adoc

```diff
@@ -90,6 +91,7 @@ image::encrypt.png[]
 |*doc:name* |  |A display name for the element in Studio's Visual Editor. Not applicable for Standalone.
 |*using* |x |JCE_ENCRYPTER
 |*input-ref* |  |Enter a Mule expression to define the part(s) of the message Mule should encrypt. If no value is entered, Mule encrypts the entire message payload.
+| *streamName* |  |
```

Comment:
```suggestion
| *streamName* | name of the decrypted payload in a PGP encrypted file. In case a file is encrypted using Mule, this shall help the user keep the original filename so that it is recovered on decryption of the resulting PGP file |
```